examples/radiation_CAJOE/radiation_CAJOE.cpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file main.cpp
+ * @author Antonio Vanegas @hpsaturn
+ * @date June 2018 - 2022
+ * @brief Radiation sensor example
+ * @license GPL3
+ * 
+ * Full documentation:
+ * https://github.com/kike-canaries/canairio_sensorlib#canairio-air-quality-sensors-library
+ * 
+ * Full implementation for WiFi and Bluetooth Air Quality fixed and mobile station:
+ * https://github.com/kike-canaries/canairio_firmware#canairio-firmware
+ * 
+ * CanAirIO project:
+ * https://canair.io
+ * 
+ * CanAirIO Docs:
+ * https://canair.io/docs
+ * 
+ */
+
+#include <Arduino.h>
+#include <Sensors.hpp>
+
+void onSensorDataOk() {
+    Serial.print(" CO2: " + String(sensors.getCO2()));
+    Serial.print(" CO2humi: " + String(sensors.getCO2humi()));
+    Serial.print(" CO2temp: " + String(sensors.getCO2temp()));
+
+    Serial.print(" H: " + String(sensors.getHumidity()));
+    Serial.println(" T: " + String(sensors.getTemperature()));
```

Comment:
The Geiger sample should have the calls regarding Geiger methods. Please change it

Reply:
done.